src/universal/modules/teamDashboard/components/Team/Team.js

```diff
@@ -65,17 +66,29 @@ const settingsLinks = (teamId) => {
 const initialValues = {teamName: ''};
 
 const Team = (props) => {
-  const {children, router, team, teamMembers} = props;
+  const {children, hasNotificationBar, router, team, teamMembers} = props;
   const {id: teamId, name: teamName, isPaid} = team;
   const hasActiveMeeting = Boolean(team && team.meetingId);
   const hasOverlay = hasActiveMeeting || !isPaid;
   const isSettings = router.isActive(`/team/${teamId}/settings`, false);
   initialValues.teamName = teamName;
   const DashHeaderInfoTitle = isSettings ? <EditTeamName initialValues={initialValues} teamName={teamName} teamId={teamId}/> : teamName;
+  const modalLayout = hasNotificationBar ? ui.modalLayout[1] : ui.modalLayout[0];
```

Comment:
referencing the array isn't very meaningful (i know i set a bad example with the notifications varList, but that's mainly for payload size). could we instead say something like `hasNotificationBar ? ui.main : ui.mainHasNotificaitonBar